_posts/2019-06-06-NeuralODEs.markdown

```diff
@@ -0,0 +1,388 @@
+---
+layout: post
+title:  "Neural ODEs"
+date:   2019-06-06 10:00:00 -0400
+categories: neural-nets
+author: luca
+blurb: "Neural ODEs are neural network models which generalize standard layer to layer propagation to continuous depth models. Starting from the observation that the forward propagation in neural networks is equivalent to one step of discretation of an ODE, we can construct and efficiently train models by via ODEs. On top of providing a novel family of architectures, notably for invertible density models and continuous time series, neural ODEs also provide a memory efficiency gain in supervised learning tasks. In this curriculum, we will go through all the background topics necessary to understand these models. At the end, you should be able to implement neural ODEs and apply them to different tasks."
+feedback: true
+---
+
+This guide would not have been possible without the help and feedback from many people. 
+
+Special thanks to Prof. Joan Bruna and his class at NYU, [Mathematics of Deep Learning](https://github.com/joanbruna/MathsDL-spring19), and to Cinjon Resnick, who introduced me to DFL and helped me putting up this guide.
```

Comment:
“and helped me putting up this guide” —> “and helped complete this guide.”

Reply:
fixed